analysis/combiners.py

```diff
@@ -320,6 +317,22 @@ def create_accumulator(
         return super().create_accumulator(data)
 
 
+class StatisticsCombiner(UtilityAnalysisCombiner):
+    """A combiner for computing per-partition statistics (count etc)."""
```

Comment:
We already have other combiners for computing per-partition metrics, no? How is this different from the others? I believe having more descriptive documentation here would help.

Reply:
We don't always compute both count/privacy_id_count, e.g. when metrics = [COUNT], privacy_id_count is not computed, that's why `StatisticsCombiner` was introduced.

But, it seems to be much details for this class docstring.